src/proxy-ui-api/frontend/src/store/modules/user.ts

```diff
@@ -121,6 +131,20 @@ export const actions: ActionTree<UserState, RootState> = {
       });
   },
 
+  async fetchCurrentSecurityServer({ commit }) {
+    return axios.get<SecurityServer[]>('/security-servers?current_server=true')
+      .then((resp) => {
+        if (resp.data?.length !== 1) {
+          throw new Error(`Failed to set current security server. Expected the response to contain 1 item, actual was ${resp.data?.length}`);
```

Comment:
The error text should be localised?
clients.ts has an example how to use i18n  in vuex.

Reply:
Added localization.